build/build.js

```diff
@@ -19,3 +20,4 @@ fs.writeFileSync(path + "promiscuous-browser.js", browser);
 fs.writeFileSync(path + "promiscuous-browser-full.js", browserFull);
 fs.writeFileSync(path + "promiscuous-shim.js",
                  browser.replace("!", "(typeof Promise)[0]!='u'||"));
+fs.writeFileSync(path + "promiscuous-browser-shim-full.js", browserShimFull);
```

Comment:
Missing newline at the end of the file, can you change and amend?

Reply:
Done.